src/routes/signup/+page.svelte

```diff
@@ -11,56 +15,29 @@
 <p>Please input your credentials</p>
 
 <form action="?/signup" method="POST">
-	<div>
+	<div class="content">
 		<label for="username">Name:</label>
-		<input type="text" id="username" name="username" required />
+		<TextInput placeholder="E-Mail Adress" type="text" id="username" name="username" required="{true}"></TextInput>
 	</div>
 
-	<div>
+	<div class="content">
 		<label for="password">Password:</label>
-		<input type="password" id="password" name="password" required />
+		<TextInput placeholder="Password" type="password" id="password" name="password" required="{true}"></TextInput>
 	</div>
 
-	{#if form?.user}
+	{#if form?.user || form?.invalid}
 		<p class="error">{form?.info}</p>
 	{/if}
 
-	<div class="btn">
-		<button type="submit">Sign Up</button>
-	</div>
+	<Button label="Sign Up" button_type="submit" type="confirm" onclick="{() => null}"/>
 </form>
 
-<!-- back-button -->
-<a class="btn" href="/">Back</a>
-
-<style>
-	form {
-		width: 100%;
-	}
-
-	input {
-		color: #b1b1b1;
-		width: auto;
-		padding: 12px 20px;
-		margin: 8px 0;
-		box-sizing: border-box;
-		font-size: 24px;
-		align-items: left;
-	}
-
-	input[type='password'] {
-		width: 100%;
-	}
-
-	:global(.touched:invalid) {
-		border-color: red;
-		outline-color: red;
-	}
+<Divider />
 
-	.btn {
-		border: 1px solid gray;
-	}
+<LinkButton destination="/login" label="Login here!"/>
+<LinkButton destination="/" label="Back"/>
 
+<style>
 	.error {
 		color: red;
 	}
```

Comment:
see above for color.

This is used on multiple pages and should therefore rather be in the global stylesheet.